.travis.yml

```diff
@@ -2,7 +2,7 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode7.3
```

Comment:
So maybe try this patch and see if it doesn't work.

http://llvm.org/viewvc/llvm-project?view=revision&revision=219009